From 2048727a901ef7d9d3803c9206730d11b2fd6ebf Mon Sep 17 00:00:00 2001
From: Jakub Zawadzki <darkjames@darkjames.pl>
Date: Sat, 7 Dec 2013 23:37:23 +0100
Subject: [PATCH] Add names for DLT_NETLINK

Before patch:
  tcpdump: listening on nlmon0, link-type 253, capture size 65535 bytes

After patch:
  tcpdump: listening on nlmon0, link-type NETLINK (Linux netlink), capture size 65535 bytes
---
 pcap.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pcap.c b/pcap.c
index 626938c..faa9762 100644
--- a/pcap.c
+++ b/pcap.c
@@ -1203,6 +1203,7 @@ static struct dlt_choice dlt_choices[] = {
 	DLT_CHOICE(DLT_NETANALYZER_TRANSPARENT, "Ethernet with Hilscher netANALYZER pseudo-header and with preamble and SFD"),
 	DLT_CHOICE(DLT_IPOIB, "RFC 4391 IP-over-Infiniband"),
 	DLT_CHOICE(DLT_DBUS, "D-Bus"),
+	DLT_CHOICE(DLT_NETLINK, "Linux netlink"),
 	DLT_CHOICE_SENTINEL
 };
 
-- 
2.4.3

